sup3r/pipeline/forward_pass.py

```diff
@@ -633,6 +636,8 @@ def __init__(self, file_paths, model_kwargs, fwp_chunk_shape,
         overwrite_cache : bool
             Whether to overwrite cache files storing the computed/extracted
             feature data
+        overwrite_ti_cache : bool
```

Comment:
Why wouldnt this just use the `overwrite_cache` arg? Is there any confusing and/or logic here?

Reply:
There are lots of times I dont want to overwrite the time index but do for the other cache.